react/week1/todo/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+import ToDoList from './ToDoList';
+import ToDoItem from './ToDoItem';
+const toDoThing = [
+  { description: "get out of bed", deadline: "wed sep 13 2017" },
+  { description: "Brush Teeth", deadline: "Thu sep 13 2017" },
+  { description: "Eat breakfast", deadline: "wed sep 13 2017" },
+];
+function App() {
+  return (
+    <div className="App">
+   <ToDoList/>
```

Comment:
Generally components are named after what they are. Looking at how your TodoList is implemented, it appears to be rendering only an H3. 

Do you think we could rename this to something more appropriate?